### PR TITLE
Update search-for-content.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the page to remove references to SFB

Fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10549

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/search-for-content.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ ms.collection:
 
 # Search for content using the Content search tool
 
-Use the Content search tool in the Microsoft Purview compliance portal to quickly find email in Exchange mailboxes, documents in SharePoint sites and OneDrive locations, and instant messaging conversations in Skype for Business. You can use the content search tool to search for email, documents, and instant messaging conversations in collaboration tools such as Microsoft Teams and Microsoft 365 Groups.
+Use the Content search tool in the Microsoft Purview compliance portal to quickly find email in Exchange mailboxes, documents in SharePoint sites and OneDrive locations. You can use the content search tool to search for email, documents, and instant messaging conversations in collaboration tools such as Microsoft Teams and Microsoft 365 Groups.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/search-for-content.md:23]:
```suggestion
Use the Content search tool in the Microsoft Purview compliance portal to quickly find email in Exchange mailboxes and documents in SharePoint sites and OneDrive locations. You can use the content search tool to search for email, documents, and instant messaging conversations in collaboration tools such as Microsoft Teams and Microsoft 365 Groups.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by cabailey** [comment]
Retention policies continue to support this retired service for existing customers - see https://learn.microsoft.com/microsoft-365/compliance/retention-settings#configuration-information-for-skype-for-business  

Retaining without the ability to search doesn't seem to make much sense?  Deleting the reference when it was previously published would infer it's no longer supported & not sure if that was checked before creating this PR.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by msbemba** [comment]
@cabailey Thank you for the confirmation. I will let the customer know.

Thanks
Sri

**Pull request closed without merging**